doc/source/visualizing/plots.rst

```diff
@@ -1054,6 +1055,68 @@ change the property of a single line, give also the index of the profile.
     # change only the first line
     plot.set_line_property("linestyle", "--", 0)
 
+.. _how-to-1d-unstructured-mesh:
+
+1D Sampling on Unstructured Meshes
+----------------------------------
+
+YT has the ability to sample unstructured mesh data-sets along arbitrary lines
+and plot the result. You must supply five arguments to the ``LinePlot``
+class. They are enumerated below:
+
+1. Data-set
+2. A list of fields or a single field you wish to plot
+3. The starting point of the sampling line. This should be a tuple of three
+   floats corresponding to the coordinates of the starting point
+4. The ending point of the sampling line. This should also be a tuple of three
+   floats
+5. The resolution of the sampling line. This is the number of sampling points
+   along the line, e.g. if 1000 is specified, then data will be sampled at
+   1000 points evenly spaced between the starting and ending points.
+
+The below code snippet illustrates how this is done:
+
+.. python-script::
+
+   ds = yt.load(home + "/yt_data/SecondOrderTris/RZ_p_no_parts_do_nothing_bcs_cone_out.e", step=-1)
+   ln = yt.LinePlot(ds, [('all', 'v'), ('all', 'u')], (0, 0, 0), (0, 1, 0), 1000)
+   ln.save("first_test.png")
+
+You can also add plots to existing ``LinePlot`` instances with ``add_plot`` as shown
+below:
+
+.. python-script::
+
+   ln.add_plot(('all', 'p'), (0, 0, 0), (1, 1, 0), 1000)
+   ln.save("added_plot.png")
+
+Note that the beginning and end-points of multiple plots can be different. If
+working in an IPython Notebook, ``LinePlot`` also has the ``show()`` method.
+
+You can also create legends and add x and y axes labels to these 1D sampling
+plots. The legend process takes two steps:
+
+1. When instantiating the ``LinePlot`` or invoking ``add_plot`` pass a dictionary of
+   labels with keys corresponding to the field names
+2. Call the ``LinePlot`` ``add_legend`` method
+
+X- and Y- axis labels are set simply with ``set_xlabel`` and ``set_ylabel``
+methods. The below code snippet combines all the features we've discussed:
+
+.. python-script::
+
+   import yt
+   ds = yt.load("SecondOrderTris/RZ_p_no_parts_do_nothing_bcs_cone_out.e", step=-1)
+   ln = yt.LinePlot(ds, [('all', 'v'), ('all', 'u')], (0, 0, 0), (0, 1, 0), 1000,
+                    labels={('all', 'u') : r"u$_s$", ('all', 'v') : r"v$_s$"})
+   ln.add_plot([('all', 'v'), ('all', 'u')], (0, 0, 0), (1, 1, 0), 1000,
+               labels={('all', 'u') : r"u$_l$", ('all', 'v') : r"v$_l$"})
+   ln.add_legend()
+   ln.set_xlabel("Arc Length (cm)")
+   ln.set_ylabel(r"Velocity (m s$^{-1}$)")
+   ln.save("line_plot.eps")
```

Comment:
If you're using `python-script` please save png images otherwise they won't show up inline in the docs.